test-fuzz/tests/integration/ci.rs

```diff
@@ -28,6 +29,27 @@ fn dylint() {
         .success();
 }
 
+#[test]
+fn cargo_sort() {
+    Command::new("cargo")
+        .args(["install", "cargo-sort"])
+        .logged_assert()
+        .success();
+
+    for entry in WalkDir::new("..")
+        .into_iter()
+        .filter_map(Result::ok)
+        .filter(|e| e.file_name().to_str() == Some("Cargo.toml"))
```

Comment:
```suggestion
        .filter(|e| e.file_name() == Some(OsStr::new("Cargo.toml")))
```
Nit. I think you will have to add the import for `std::ffi::OsStr` at the top of the file.